webviz_subsurface/containers/_intersect.py

```diff
@@ -0,0 +1,297 @@
+                    ),
+                    html.P('Realization:', style={'font-weight': 'bold'}),
+                    dcc.Dropdown(
+                        id='real_list',
+                        options=[{'label': real, 'value': path}
+                                 for path, real in self.realizations.items()],
+                        value=list(self.realizations.keys())[0],
+                        multi=True,
+                        placeholder='All realizations'
+                    ),
+
+                    html.P('Surfaces:', style={'font-weight': 'bold'}),
+                    dcc.Dropdown(
+                        id='surf_list',
+                        options=[{'label': r, 'value': r}
+                                 for r in self.surface_names],
+                        value=self.surface_names[0],
+                        multi=True,
+                        placeholder='All surfaces'
+                    ),
+                    html.Div(style=self.table_layout, children=[
+                        DataTable(
+                            id='table',
+                            columns=[{"name": i, "id": i}
+                                     for i in ['Name', 'TVDmin', 'TVDmean',
+                                               'TVDmax', 'TVDstddev']]
+                        )
+                    ])
+                ]),
+                html.Div(style={'height': '80vh'}, children=[
+                    dcc.Graph(style={'height': '80vh'}, id='graph'),
+                    html.Div(style=self.style_fence_options, children=[
+                        html.P('Start depth:', style={'font-weight': 'bold'}),
+                        html.P('Graph zoom level:', style={
+                               'font-weight': 'bold'}),
+                        dcc.Input(style={'overflow': 'auto'},
+                                  id='well-tvd', type='number', value=0),
+                        dcc.RadioItems(
+                            id='ui_revision',
+                            options=[{'label': k, 'value': v}
+                                     for k, v in {
+                                     'Keep on new data': True,
+                                     'Reset on new data': False}.items()],
+                            value=True,
+                        ),
+                    ])
+
+                ])
+            ])
+        ])
+
+    def set_callbacks(self, app):
+        @app.callback(
+            Output('graph', 'figure'),
+            [Input('well_list', 'value'),
+             Input('real_list', 'value'),
+             Input('surf_list', 'value'),
+             Input('well-tvd', 'value'),
+             Input('ui_revision', 'value')])
+        def set_fence(_well_path, _reals, _surfs, _tvdmin, _keep_zoom_state):
+            if not isinstance(_surfs, list):
+                _surfs = [_surfs]
+            if not isinstance(_reals, list):
+                _reals = [_reals]
+            if _reals == []:
```

Comment:
[`if not _reals:`](https://stackoverflow.com/questions/53513/how-do-i-check-if-a-list-is-empty)